### PR TITLE
Polish translation improved

### PR DESCRIPTION
**Pull request by michalciolek**

### CODE REVIEW

File: po/pl.po [hunk 1]
```diff
@@ -4499,19 +4492,16 @@ msgid "_Pay Invoice"
...
-#, fuzzy
 msgid "_Edit Bill"
-msgstr "Edytuj rachunek"
+msgstr "E_dytuj rachunek"
 
 #: gnucash/gnome/gnc-plugin-page-invoice.c:280
-#, fuzzy
 msgid "_Duplicate Bill"
-msgstr "Zduplikuj"
+msgstr "Duplikuj rachunek"
```
Comment (fellen) [inline po/pl.po]: This is one of 15 places, where the accelerator is missing. The full output from `msgfmt -c --check-accelerators="_" --statistics pl.po` is

> pl.po:3783: msgstr lacks the keyboard accelerator mark '_'
> pl.po:3801: msgstr lacks the keyboard accelerator mark '_'
> pl.po:4504: msgstr lacks the keyboard accelerator mark '_'
> pl.po:5136: msgstr lacks the keyboard accelerator mark '_'
> pl.po:8547: msgstr lacks the keyboard accelerator mark '_'
> pl.po:12320: msgstr lacks the keyboard accelerator mark '_'
> pl.po:12334: msgstr lacks the keyboard accelerator mark '_'
> pl.po:16365: msgstr lacks the keyboard accelerator mark '_'
> pl.po:16945: msgstr lacks the keyboard accelerator mark '_'
> pl.po:17272: msgstr lacks the keyboard accelerator mark '_'
> pl.po:17276: msgstr lacks the keyboard accelerator mark '_'
> pl.po:17448: msgstr lacks the keyboard accelerator mark '_'
> pl.po:20166: msgstr lacks the keyboard accelerator mark '_'
> pl.po:21146: msgstr lacks the keyboard accelerator mark '_'
> pl.po:25425: msgstr lacks the keyboard accelerator mark '_'
> msgfmt: found 15 fatal errors
> 3181 translated messages, 1540 fuzzy translations, 685 untranslated messages.
https://wiki.gnucash.org/wiki/Translation#Check_syntax_and_statistics_of_your_.po_file

File: po/pl.po [hunk 2]
```diff
@@ -4,8 +4,8 @@ msgstr ""
 "Report-Msgid-Bugs-To: https://bugs.gnucash.org/enter_bug.cgi?"
 "product=GnuCash&component=Translations\n"
 "POT-Creation-Date: 2019-12-26 16:42+0100\n"
```
Comment (fellen) [inline po/pl.po]: On which git version is your work based? cstim applied a few changes in recent days.

Comment (michalciolek) [reply po/pl.po]: The latest one, which contains cstim's changes.

File: po/pl.po (hunk 1 above)
Comment (michalciolek) [reply po/pl.po]: I know, I know... there are so many works... not for one, two or even three days of work. I will be adding corrections successively. I will let you know when I'm done and you can merge the changes.

**Comment by michalciolek** [comment]
> Hi Michał,
> nice work. I made a few annotations inline.
> 
> Do you have time to improve this file more? There seems to be still 1540 fuzzy translations, 685 untranslated messages.

I have some time, I will try to improve slowly. I can't guarantee I will improve everything.

I installed http://weblate.org/ locally (using Docker). This is a great tool and I suggest using it for gnucash. It's free for opensource. It works based on the browser, updates are included using GihtHub PR, etc.

**Comment by fellen** [comment]
> PS. I installed http://weblate.org/ locally (using Docker) to improve translation. This is a great tool and I suggest using it for gnucash. It's free for opensource. It works based on the browser, updates are included using GitHub PR, etc.

Can you add a section to https://wiki.gnucash.org/wiki/Improve_Localization_Process about the setup?

**Comment by fellen** [comment]
Another note:
We have a few new terms in the glossary. You  should consider to update it:
https://wiki.gnucash.org/wiki/Translation#The_glossary_file

**Comment by fellen** [comment]
> I have some time, I will try to improve slowly. I can't guarantee I will improve everything.

OK, I have applied this part. Additionally I have recovered the older contributors into translator credit and merged a current gnucash.pot. So you should now rebase before you push your next PR.

Thank you

**Pull request merged**